app/uk/gov/hmrc/tariffclassificationfrontend/views/confirm_complete_case.scala.html

```diff
@@ -18,24 +18,30 @@
 @import uk.gov.hmrc.tariffclassificationfrontend.models.{BTIApplication, Case}
 @import uk.gov.hmrc.tariffclassificationfrontend.views.html.includes.main
 @import uk.gov.hmrc.tariffclassificationfrontend.views.html.partials._
+@import uk.gov.hmrc.tariffclassificationfrontend.models.ApplicationType
 @(c: Case)(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
 
 @main(bodyClasses = None, customTitle = Some(messages("page.title.case.completed"))) {
-    @defining(c.application.asBTI) { btiApp: BTIApplication =>
 
-        @case_heading(c)
+    @case_heading(c)
 
-        <div class="grid-row">
-            <div class="column-full">
+    <div class="grid-row">
+        <div class="column-full">
+            @if(c.application.`type` == ApplicationType.BTI) {
                 <h2 class="heading-large mt-0">This case has been completed</h2>
 
                 <p>The ruling has been published.</p>
                 <p>View the published ruling on the public rulings website.</p>
                 <p>The applicant has been notified by email.</p>
                 <p class="bold">The status of this case is now completed</p>
+            } else {
+                <h2 class="heading-large mt-0">This liability decision is complete</h2>
 
-                @confirmation_next_steps()
-            </div>
+                <p>Email the applicant with a copy of tthis decision.</p>
```

Comment:
Think Aimee pointed out this typo on 'tthis' - did you fix it?

Reply:
Sorted, cheers